apps/s_server.c

```diff
@@ -962,6 +964,7 @@ const OPTIONS s_server_options[] = {
     {"max_early_data", OPT_MAX_EARLY, 'n',
      "The maximum number of bytes of early data"},
     {"early_data", OPT_EARLY_DATA, '-', "Attempt to read early data"},
+    {"force_pha", OPT_FORCE_PHA, '-', "Force-enable post-handshake-authentication (violates protocol)"},
```

Comment:
Why does this violate the protocol?

Reply:
Because it allows the server to send the post-handshake authentication request without receiving the post-handshake extensions; to be used for testing.